lore/species/tajaran/The Great Frost.md

```diff
@@ -0,0 +1,85 @@
+
+In any case the tremors of his actions echo through tajaran society today. In most holds money commonly bears either the depiction of his face or the symbol of his birthmark above or on the face of rulers. Many temples have small shrines devoted towards the savior of the tajaran and it is not uncommon for prayers to thank Akari for his actions. The greatest shrines devoted towards him are within the hearth engines and frost breakers themselves, nearly all are decorated with his symbol and treated as holy objects by some.
+
+
+Side Fluff without a Category
+
+Glare of S�randarr
+The Glare of S�randarr is a curious and complex weapon developed, seemingly simultaneously, by several tajaran holds that became very widely used for their durability, easy maintenance, effectiveness, and utility. The early versions of the Glare of S�randarr were actually modified heat lamps originally built to keep Sun Wardens warm during expeditions. These heaters were modified, portable version of the hearth engines kept heated by battery-like cores charged with the hearth engines power. The idea to weaponize them came about naturally as early prototypes were altered to allow a crossbow mounting, with only a few modifications these devices could transfer exothermic energy to metal bars loaded into the crossbow mounting. Many were� clunky, slow, and took far to long to reload than was practical. However, the single shot and lethality of such ammunition did not go unnoticed and soon a push for further development began.
+
+The process was refined to guns specifically built to house the battery cores and super-heat metal pits shaped and fed through a swap-able magazine. Over time and through careful refinement these weapons over took and eventually replaced kinetic weapons among most Sun Wardens and hold border guards. A secondary advantage outside its combat ability was the simple fact it also doubled as a powerful portable heater, a useful trait for Sun Wardens traveling long distances and needing to pack lightly. The name, regardless of the design, of these weapons is the Glare of S�randarr. However a common slang has come about among those that use them to differentiate from the various model types.

```

Comment:
"... through a swappable magazine."
"... through careful refinement, these weapons overtook and eventually..."